tests/pages/roadmap-page.js

```diff
@@ -20,6 +20,17 @@ export default createPage({
     voteCountText: text('[data-test-vote-count]'),
   }),
 
+  latestReleases: {
+    scope: '[data-test-latest-releases]',
+    title: text('[data-test-latest-releases-title]'),
+    viewMoreLink: text('[data-test-latest-releases-view-more-link]'),
```

Comment:
Wrong naming if we're testing text - should be `viewMoreLinkText`. Just testing the text is kinda pointless though, either we test clicking the link to see if the destination URL was redirected to, or we don't test at all